Presentation.Web/Controllers/API/V2/External/Tokens/TokenV2Controller.cs

```diff
@@ -20,7 +21,7 @@ public class TokenV2Controller : ExternalBaseController
         [IgnoreCSRFProtection]
         public IHttpActionResult Introspect([FromBody] TokenIntrospectionRequest request)
```

Comment:
Since the models have been moved to the Core layer we should map them to DTOs.

Same thing in the V1 controller